### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by Original-Tasty**

### CODE REVIEW

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PACKAGE=qt
 $(package)_version=5.9.7
-$(package)_download_path=http://ftp.jaist.ac.jp/pub/qtproject/archive/qt/5.9/$($(package)_version)/submodules/
+$(package)_download_path=https://download.qt.io/archive/qt/5.9/$($(package)_version)/submodules
```
Comment (bmerriam) [inline depends/packages/qt.mk:3]: This was updated to use the .jp site as the qt.io does not carry the 5.9 version anymore.  I'm unsure what the impact could be by upgrading, so I used an archive location.  Thoughts appreciated.

File: depends/patches/native_cdrkit/cdrkit-deterministic.patch [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
++	//current_dir = opendir(path);
++        current_file = scandir(path, &d_list, NULL, alphasort);
+ 	d_entry = NULL;
+ 
+ 	/*
+@@ -1173,12 +1175,12 @@
+ 	 */
+ 	old_path = path;
+ 
+-	if (current_dir) {
```
Comment (bmerriam) [inline depends/patches/native_cdrkit/cdrkit-deterministic.patch:28]: Were these a merge/patch from a different version or fork?  It's a decent amount of changes, curious of you did them yourself or if we should source a newer fork for future iterations.  Thanks!

File: depends/packages/qt.mk (hunk 1 above)
Comment (bmerriam) [reply depends/packages/qt.mk:3]: checked repo, looks good!

File: depends/patches/native_cdrkit/cdrkit-deterministic.patch (hunk 2 above)
Comment (Original-Tasty) [reply depends/patches/native_cdrkit/cdrkit-deterministic.patch:28]: Hey!  no merge or patch, just researches! I've forked several times these last month Litecoin v0.18.1 to other coins so i knew where to look at :P 
Commits are a bit messy and not in order lol sorry 'bout that
Anyway thanks for taking those into consideration!

**Pull request merged**